Source/Core/Core/HW/HW.cpp

```diff
@@ -97,6 +100,8 @@ void DoState(PointerWrap& p)
   p.DoMarker("ExpansionInterface");
   AudioInterface::DoState(p);
   p.DoMarker("AudioInterface");
+  HSP::DoState(p);
+  p.DoMarker("HSP");
```

Comment:
You should increment the [state version](https://github.com/dolphin-emu/dolphin/blob/37c85b32c058e820d0bc81aca8d380e09aef1bd5/Source/Core/Core/State.cpp#L77) as well.

Reply:
This needs fixing again now.